src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -146,12 +146,19 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+//    def listMonitoredClusterOnly(): DBIO[Seq[Cluster]] = {
+//      clusterQuery.filter{_.status inSetBind ClusterStatus.monitoredStatuses.map(_.toString) }.result map { recs =>
+//        recs.map(rec => unmarshalCluster(rec, Seq.empty, List.empty, Map.empty, List.empty, List.empty, List.empty))
+//      }
+//    }
+
     def listMonitoredClusterOnly(): DBIO[Seq[Cluster]] = {
-      clusterQuery.filter{_.status inSetBind ClusterStatus.monitoredStatuses.map(_.toString) }.result map { recs =>
-        recs.map(rec => unmarshalCluster(rec,Seq.empty, List.empty, Map.empty, List.empty, List.empty, List.empty))
+      clusterImagesQuery.filter{_._1.status inSetBind ClusterStatus.monitoredStatuses.map(_.toString) }.result map { recs =>
```

Comment:
It might be easier and faster to get the to use `getAllForCluster` from `clusterImageQuery` to get the images for each cluster.